setup/docker/utils.go

```diff
@@ -19,6 +19,30 @@ func LoadConfig(path string) (*Config, error) {
 
 	if config.Auths == nil {
 		config.Auths = map[string]Auth{}
+	} else if len(config.Auths) == 0 {
```

Comment:
This behavior should exist in the [createOrWriteDockerCFG](https://github.com/quintilesims/layer0/blob/develop/setup/instance/init.go#L100) function.